packages/plugin-workspace-tools/sources/commands/foreach.ts

```diff
@@ -104,6 +105,11 @@ export default class WorkspacesForeachCommand extends BaseCommand {
     description: `Avoid running the command on private workspaces`,
   });
 
+  since = Option.String(`--since`, {
+    description: `Only include packages that have been changed since the specified ref. If no ref is passed, it defaults to the default branch.`,
```

Comment:
Options need to have short descriptions; details (like default behaviour when no values are specified, etc) should be added to the `details` field of the usage.

```suggestion
    description: `Only include packages that have been changed since the specified ref`,
```

Reply:
Fixed in 0fcfcbea37bbd13b1d08ffd1359ce8fc12d96692.